services/galley/test/integration/API/Teams/LegalHold.hs

```diff
@@ -107,37 +112,63 @@ testSwaggerJsonConsistency = do
+        do userStatus <- getUserStatusTyped member tid
+           liftIO $ assertEqual "User legal hold status should start as disabled" userStatus UserLegalHoldDisabled
 
-    do requestDevice owner member tid !!! const 200 === statusCode
-       userStatus <- getUserStatusTyped member tid
-       liftIO $ assertEqual "requestDevice should set user status to Pending"
-                  userStatus UserLegalHoldPending
+        do requestDevice member member tid !!! const 403 === statusCode
+           userStatus <- getUserStatusTyped member tid
+           liftIO $ assertEqual "User with insufficient permissions should be unable to change user status"
+                      userStatus UserLegalHoldDisabled
 
-    do requestDevice owner member tid !!! const 200 === statusCode
-       userStatus <- getUserStatusTyped member tid
-       liftIO $ assertEqual "requestDevice when already pending should leave status as Pending"
-                  userStatus UserLegalHoldPending
+        do requestDevice owner member tid !!! const 204 === statusCode
+           userStatus <- getUserStatusTyped member tid
+           liftIO $ assertEqual "requestDevice should set user status to Pending"
+                      userStatus UserLegalHoldPending
 
-    cassState <- view tsCass
-    liftIO $ do
-        storedPrekeys <- Cql.runClient cassState (LegalHoldData.selectPendingPrekeys member)
-        assertBool "user should have pending prekeys stored" (not . null $ storedPrekeys)
+        do requestDevice owner member tid !!! const 204 === statusCode
+           userStatus <- getUserStatusTyped member tid
+           liftIO $ assertEqual "requestDevice when already pending should leave status as Pending"
+                      userStatus UserLegalHoldPending
+
+        cassState <- view tsCass
+        liftIO $ do
+            storedPrekeys <- Cql.runClient cassState (LegalHoldData.selectPendingPrekeys member)
+            assertBool "user should have pending prekeys stored" (not . null $ storedPrekeys)
+
+        -- Run twice since we requested a device twice above
+        liftIO $ do
+            void . liftIO $ WS.assertMatch (5 WS.# WS.Second) ws $ \n -> do
+                let j = Aeson.Object $ List1.head (ntfPayload n)
+                let etype = j ^? key "type" . _String
+                let eRequester = j ^? key "requester" . _String
+                let eTargetUser = j ^? key "target_user" . _String
+                let eLastPrekey = j ^? key "last_prekey" . _JSON
+                let ePrekeys = j ^? key "prekeys" . _JSON
+                etype @?= Just "user.client-legal-hold-request"
+                eRequester @?= Just (idToText owner)
+                eTargetUser @?= Just (idToText member)
+                -- These need to match the values provided by the 'dummy service'
+                Just (lastPrekey "test-last-prekey") @?= eLastPrekey
+                Just [Prekey (PrekeyId 0) "test-prekey"] @?= ePrekeys
```

Comment:
maybe cross-ref the two identical tests in brig, galley?  (i still think it's a good idea to have them both.)